### PR TITLE
Update index.md - SDK section in docs

### PR DESCRIPTION
**Pull request by edgargarciamacrometa**
Error handling fixed using catch method from promises.

### CODE REVIEW

File: docs/sdks/index.md [hunk 1]
```diff
@@ -77,131 +77,171 @@ Once the installation process is finished, you can begin developing applications
...
-        console.log("Clearing Access Level for Collection Failed: ",e)
-
-    }
-    // Remove an api key
-    try{
-        await client.removeApiKey(keyid);
-    }
-    catch(e){
-        console.log('API Deletion Failed: ', e);
+// Email and Password to Authenticate client instance
```
Comment (diana-macrometa) [inline docs/sdks/index.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/sdks/index.md [hunk 2]
```diff
@@ -77,131 +77,171 @@ Once the installation process is finished, you can begin developing applications
...
+const streamName = "testStream";
+const keyid = "id1";
+
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
 
-    }
+// Or Create an authenticated instance with a token or API key.
```
Comment (diana-macrometa) [inline docs/sdks/index.md]:
```suggestion
// Or create an authenticated instance with a token or API key.
```

File: docs/sdks/index.md [hunk 3]
```diff
@@ -77,131 +77,171 @@ Once the installation process is finished, you can begin developing applications
...
+    .login(email, password)
+    .then((e) => console.log("User authentication done!"))
+    .catch((error) => error);
+  console.log("1. Creating API Key with KeyID = " + keyid);
+  await client
+    .createApiKey(keyid)
+    .then((apiKey) => console.log(apiKey))
+    .catch((error) => messageHandler(error));
+
+  console.log("\n2. Getting available API Keys");
```
Comment (diana-macrometa) [inline docs/sdks/index.md]:
```suggestion
  console.log("\n2. Getting available API keys");
```

File: docs/sdks/index.md [hunk 4]
```diff
@@ -77,131 +77,171 @@ Once the installation process is finished, you can begin developing applications
...
+  console.log(message);
 }
 
-main();
+async function main () {
+  await client
+    .login(email, password)
+    .then((e) => console.log("User authentication done!"))
+    .catch((error) => error);
+  console.log("1. Creating API Key with KeyID = " + keyid);
```
Comment (diana-macrometa) [inline docs/sdks/index.md]:
```suggestion
  console.log("1. Creating API key with KeyID = " + keyid);
```

File: docs/sdks/index.md [hunk 5]
```diff
@@ -77,131 +77,171 @@ Once the installation process is finished, you can begin developing applications
...
+
+  console.log("\n3. Listing accessible databases for Key_ID = " + keyid);
+  await client
+    .listAccessibleDatabases(keyid)
+    .then((databases) => {
+      console.log(databases.result);
+    })
+    .catch((error) => messageHandler(error));
+
+  console.log("\n4. Listing accesible Streams for Key_ID = " + keyid);
```
Comment (diana-macrometa) [inline docs/sdks/index.md]:
```suggestion
  console.log("\n4. Listing accessible streams for Key_ID = " + keyid);
```

File: docs/sdks/index.md [hunk 6]
```diff
@@ -77,131 +77,171 @@ Once the installation process is finished, you can begin developing applications
...
+  await client
+    .listAccessibleStreams(keyid, fabric, (full = false))
+    .then((streams) => {
+      console.log(streams.result);
+    })
+    .catch((error) => messageHandler(error));
+
+  console.log("\n5. Listing Collections");
```
Comment (diana-macrometa) [inline docs/sdks/index.md]:
```suggestion
  console.log("\n5. Listing collections");
```

File: docs/sdks/index.md [hunk 7]
```diff
@@ -77,131 +77,171 @@ Once the installation process is finished, you can begin developing applications
...
+  await client
+    .getCollections()
+    .then((collections) => {
+      console.log(collections);
+    })
+    .catch((error) => messageHandler(error));
+
+  console.log("\n6. Creating Collection");
```
Comment (diana-macrometa) [inline docs/sdks/index.md]:
```suggestion
  console.log("\n6. Creating collection");
```

File: docs/sdks/index.md [hunk 8]
```diff
@@ -77,131 +77,171 @@ Once the installation process is finished, you can begin developing applications
...
+  await client
+    .createCollection(collectionName)
+    .then((collection) => {
+      console.log("Collection Created Successfully");
```
Comment (diana-macrometa) [inline docs/sdks/index.md]:
```suggestion
      console.log("Collection created successfully");
```

File: docs/sdks/index.md [hunk 9]
```diff
@@ -77,131 +77,171 @@ Once the installation process is finished, you can begin developing applications
...
+
+  console.log("\n7. Setting collection access level");
+  await client
+    .setCollectionAccessLevel(keyid, fabric, collectionName, "rw")
+    .then((collectionAccessLevel) => {
+      console.log(collectionAccessLevel);
+    })
+    .catch((error) => messageHandler(error));
+
+  console.log("\n8. Creating Stream " + streamName);
```
Comment (diana-macrometa) [inline docs/sdks/index.md]:
```suggestion
  console.log("\n8. Creating stream " + streamName);
```

File: docs/sdks/index.md [hunk 10]
```diff
@@ -77,131 +77,171 @@ Once the installation process is finished, you can begin developing applications
...
+  await client
+    .createStream(streamName)
+    .then((stream) => console.log(stream))
+    .catch((error) => messageHandler(error));
+
+  console.log(
+    "\n9. Setting Stream " + streamName + " access level to read only"
```
Comment (diana-macrometa) [inline docs/sdks/index.md]:
```suggestion
    "\n9. Setting stream " + streamName + " access level to read only"
```

File: docs/sdks/index.md [hunk 11]
```diff
@@ -77,131 +77,171 @@ Once the installation process is finished, you can begin developing applications
...
+  );
+  await client
+    .setStreamAccessLevel(keyid, fabric, "c8globals." + streamName, "ro")
+    .then((streamAccessLevel) => console.log(streamAccessLevel))
+    .catch((error) => messageHandler(error));
+
+  console.log(
+    "\n10. Setting Database " + fabric + " access level to read write for Key_ID " +
```
Comment (diana-macrometa) [inline docs/sdks/index.md]:
```suggestion
    "\n10. Setting database " + fabric + " access level to read write for Key_ID " +
```

**Review by diana-macrometa — APPROVED** [review]
Minor edits, then good to go.

**Review by jamesm-macrometa — APPROVED** [review]
LGTM!

**Pull request merged**